js/dataframe/shared/Table.svelte

```diff
@@ -263,6 +263,7 @@
 			switch (event.key) {
 				case "ArrowDown":
 					selected = [0, selected_header];
+					selected_cells = [[0, selected_header]];
```

Comment:
There was an issue where if you were in the topmost row and you pressed the up key, it would keep the topmost cell selected while also selecting the header cell. These arrow-related changes fix that while also allowing you to go "back down" from the header cell to the cell value.

https://github.com/user-attachments/assets/4df2d041-ab57-4943-ac70-e46ebad65403